pkg/volume/iscsi/iscsi.go

```diff
@@ -23,7 +23,7 @@ import (
 	"strconv"
 	"strings"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Just curious, why the import v1 as v1 here?

Reply:
This was added by goimports. They say it's intended: https://github.com/golang/go/issues/30051